examples/basic_usage.py

```diff
@@ -295,6 +295,20 @@ def my_func(x):
 # %% [markdown]
 # _Stay tuned for some new features!_
 
+# %% [markdown]
+# ---
+# # Cross-validation with skore
+
+# %%
+from sklearn import datasets, linear_model
+from skore.cross_validate import cross_validate
+diabetes = datasets.load_diabetes()
+X = diabetes.data[:150]
+y = diabetes.target[:150]
+lasso = linear_model.Lasso()
+
+cv_results = cross_validate(lasso, X, y, cv=3)
```

Comment:
Question: there's no need to specify the `project`?

I'm asking because the examples before do save something to the project, so I thought it'd be more consistent to specify it.

Reply:
I was mostly using this example to develop the plot, so I didn't need to save stuff in the project -- I'll add it.